test/dotnet-add-p2p.Tests/GivenDotnetAddP2P.cs

```diff
@@ -52,6 +54,20 @@ private ProjDir NewLib([System.Runtime.CompilerServices.CallerMemberName] string
             return dir;
         }
 
+        private static void SetTargetFrameworks(ProjDir proj, string[] frameworks)
```

Comment:
this logic is repeated in the tests - this is because @jgoshi is planning to do refactoring on some of these and it will make merge conflict easier